src/pages/using-spark/components/highlight-board.mdx

```diff
@@ -94,4 +94,6 @@ a solid background color, or no image.
 - When using a Highlight Board that includes an image,
 be sure to include a descriptive `alt` attribute.
 - When using a solid background color, the contrast ratio
-between the headline and the background must be at least 3:1.
\ No newline at end of file
+between the headline and the background must be at least 3:1.
+- To check the contrast ratio of your text on a background image,
+check out [this resource](https://www.brandwood.com/a11y/).
```

Comment:
Everything looks great, I would just like for this link, for you to sync with Olya and see if she has one for us.